src/info.c

```diff
@@ -25,13 +25,16 @@
 enum swupd_code print_update_conf_info()
 {
 	enum swupd_code ret = SWUPD_OK;
-
+	char dist_string[LINE_MAX];
 	int current_version = get_current_version(path_prefix);
-	if (current_version < 0) {
+	bool dist_string_found = get_distribution_string(path_prefix, dist_string);
+
+	if (current_version < 0 || !dist_string_found) {
```

Comment:
If !dist_string_found you have the information about the version to display. So instead you can print

    Distribution: unknown
    Installed version: 12345

And if current_version < 0

    Error: Unable to determine current OS version
    Distribution: Clear Linux OS
    Installed version: unknown